backend/api.py

```diff
@@ -77,6 +76,15 @@ def Return_Laundry():
         ]
         return jsonify(data)
 
+@app.route('/dummy/userSignUp', methods=['GET'])
+def Return_New_User():
+    if request.method == 'GET':
+        newUser = models.User()
+        db.session.add(newUser)
+        db.session.commit()
+        users = models.User.query.all()
+        return {"userId": len(users)}
```

Comment:
This is not a safe/consistent way to get the `userId`. all you need is to return `{'userId': newUser.id}`